CHANGES/1106.misc

```diff
@@ -0,0 +1 @@
+check every commit for jira url and changelog updates
```

Comment:
Minor... maybe this should be ~"Update method to check commit for jira url and changelog updates"?